### DISCUSSION TITLE
2022 年终总结 | liruifengv

### DISCUSSION THREAD

**Discussion by giscus** [liruifengv/liruifengv.com#32] [Announcements]
# 2022 年终总结 | liruifengv

2022 年是难以言表的一年，没完没了的疫情席卷着我们。所幸的是，2022 年末的时候，迎来了新的转机。抓住 12 月的尾巴，写一下今年的年终总结。

https://liruifengv.com/posts/2022-year-end-summary/

**Comment by hongtaoh** [comment]
好奇问一句，目前的 astro 博客为什么还要用 rust 做后台？我没看到需要互动的数据，仅有的评论也是外部的。
